.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Documentation
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "exclude-from-changelog"
```

Comment:
Maybe `dependencies` and/or `Infrastructure`, which are existing and used labels in Nilearn rather than `exclude-from-changelog` ?

Reply:
thanks for catching that.

should be fixed now